setup.cfg

```diff
@@ -43,7 +43,7 @@ install_requires =
     xarray >=0.19
     scipy >=1.4,!=1.6.0,!=1.6.1
     sympy >=1.6
-    pint >=0.18
+    pint ==0.18
```

Comment:
Of course, pint had to update to a new incompatible version when I think I finally got everything working. This was hilarious. The problem with the new version is that now uses full names for some reason which causes some doctests to fail. Have to figure out why and unpin the version afterward.

Reply:
I cant look into it in detail right now but I guess with 'full names' you mean the unit format?
So `meter` instead of `m`?

The use of 'short notation' is `weldx` specific for our unit registry and done here https://github.com/BAMWelDX/weldx/blob/af9df21e2efe10613847cb1d0c6f4b122bfbc880/weldx/constants.py#L29-L31

Maybe the syntax or notation has changed to set this setting

Reply:
I will take a look